stock-price-points/data/dataGenerator.js

```diff
@@ -0,0 +1,141 @@
+const fs = require('fs');
+const { companyNames, alphabet } = require('./idGenerator');
+// var wstream = fs.createWriteStream('./data/seed/dataA.json');
+
+let dataSetSize = 500000; // number of companies
+
+// generate ticker & associating data
+const generateTickerSymbol = (set) => {
+  dataSetSize = (set[0] === 'V') ? 403505 : 500000;
+  dataSetSize = (set[0] === 'W') ? 5 : 500000;
+  const result = [];
+  let count = 0;
+  const uniqueTicker = (current = '') => {
+    if (count >= dataSetSize) { return; }
+    if (current.length === 5) {
+      count++;
+      return result.push(current);
+    }
+    alphabet.forEach(char => uniqueTicker(current + char));
+  };
+  set.forEach(letter => { uniqueTicker(letter) });
+  return result;
+};
+
+// company names
```

Comment:
nice comments 👍

Reply:
Thanks! Took the suggestion & added a script file :)